src/ripple/basics/TaggedCache.h

```diff
@@ -722,17 +727,15 @@ class TaggedCache
         clock_type::time_point const& when_expire,
         clock_type::time_point const& now,
         typename KeyOnlyCacheType::map_type& partition,
-        std::vector<std::shared_ptr<mapped_type>>& stuffToSweep,
-        std::atomic<int>& allRemovals,
-        std::lock_guard<std::recursive_mutex> const& lock)
+        SweptPointersVector& stuffToSweep,
```

Comment:
Consider marking this parameter with `[[maybe_unused]]`.

Reply:
Do you mean `stuffToSweep`? Why are you suggesting this? I think it is likely to be used.

Reply:
Thanks. Is adding the `[[maybe_unused]]` attribute preferred to just removing the variable name, which is what I usually do?

Reply:
I usually just remove the variable name too.  I noticed `[[maybe_unused]]` was used in the other `sweepHelper` overload, so I reached for it.  But I'm not going to loose a lot of sleep over either choice. :-)

Reply:
Hmm... making guidelines up on the fly, which is never a good idea.  But perhaps remove the variable name if it is definitely never used, and use `[[maybe_unused]]` if it _might_ or _might_not_ be used, such as in an `assert` or under a `ifdef`.

Reply:
That makes sense to me, remove the variable name if there is no reference to it at all, and `[[maybe_unused]]` if it could be under some condition. Thanks Howard!